src/lib/i18n/locales/es-ES/translation.json

```diff
@@ -390,13 +390,13 @@
 	"Enable Memory Mapping (mmap) to load model data. This option allows the system to use disk storage as an extension of RAM by treating disk files as if they were in RAM. This can improve model performance by allowing for faster data access. However, it may not work correctly with all systems and can consume a significant amount of disk space.": "Habilitar Mapeado de Memoria (mmap) para cargar datos del modelo. Esta opción permite al sistema usar el almacenamiento del disco como una extensión de la RAM al tratar los archivos en disco como si estuvieran en la RAM. Esto puede mejorar el rendimiento del modelo al permitir un acceso más rápido a los datos. Sin embargo, puede no funcionar correctamente con todos los sistemas y puede consumir una cantidad significativa de espacio en disco.",
 	"Enable Message Rating": "Habilitar Calificación de los Mensajes",
 	"Enable Mirostat sampling for controlling perplexity.": "Algoritmo de decodificación de texto neuronal que controla activamente el proceso generativo para mantener la perplejidad del texto generado en un valor deseado. Previene las trampas de aburrimiento (por excesivas repeticiones) y de incoherencia (por generación de excesivo texto).",
-	"Enable New Sign Ups": "Habilitar Registros de Nuev@s Usuari@s",
+	"Enable New Sign Ups": "Habilitar Registros de Nuevos Usuarios",
 	"Enabled": "Habilitado",
-	"Enforce Temporary Chat": "",
+	"Enforce Temporary Chat": "Forzar el uso de Chat Temporal",
```

Comment:
Temporero instead of "temporal". That means something else in this context

Reply:
What would you use here? `Temporario`? I got `Temporal` from the equivalent feature in WhatsApp (link in Spanish: https://faq.whatsapp.com/673193694148537/?locale=es_ES)
![image](https://github.com/user-attachments/assets/724943d4-7af9-477f-9751-1a3d52a1a4be)

Reply:
"Temporal" is probably fine, since its time based

Reply:
Hi @gaby, you're suggesting to leave as it's right now in the diff?

`"Enforce Temporary Chat": "Forzar el uso de Chat Temporal",`

Reply:
Yeah, leave as is. Looks good

Reply:
Thanks for checking!